nano/node/json_handler.cpp

```diff
@@ -2009,10 +2009,21 @@ void nano::json_handler::election_statistics ()
 	unsigned hinted_count = 0;
 	unsigned optimistic_count = 0;
 	unsigned total_count = 0;
+	std::chrono::steady_clock::duration total_age{};
+	auto now = std::chrono::steady_clock::now ();
+	std::chrono::steady_clock::time_point oldest_election_start = now;
 
 	for (auto const & election : active_elections)
 	{
 		total_count++;
+		auto election_start = election->get_election_start ();
+		auto age = now - election_start;
+		total_age += age;
+		if (election_start < oldest_election_start)
```

Comment:
Can oldest_election_start = std::min(oldest_election_start, election->election_start) be used instead of branching?

Reply:
I don't think I can do that if using the getter function. That would probably require election_start to be public which I think is a bad idea. It shouldn't be modified outside the election class

Reply:
This should be compatible with the getter:

oldest_election_start = std::min(oldest_election_start, election->get_election_start())